### DISCUSSION TITLE
Why gsudo installs in "Program Files (x86)" in Windows x64?

### DISCUSSION THREAD

**Discussion by DervishD** [gerardog/gsudo#223] [Q&A] [answered]
Hi!

I'm using Windows 10, x64, Home Edition. I wanted to test `gsudo` and I installed it on my system using the MSI installer for version 2.0.4. It installs, by default, in `Program Files (x86)` instead of `Program Files`. If it is detecting the platform properly, and it is doing so because on launch the 64 version of gsudo is the one running, why does it install on the x86 version of `Program Files`?

I don't think this is a bug, I'm just curious why is `gsudo` doing this, just in case I'm doing something wrong when installing it.

Thanks!

**Comment by gerardog** [comment, marked as answer]
Sorry for answering so late. You did nothing wrong. At first I thought having one installer for all platforms was a good idea: removing the user to have to make the decision "which platform to download"... So, to have a single .MSI for all platforms I had to bind the installer to the lower denominator (x86), which worked, but forced the install to "x86" folder.
On 2.0.6 I reverted that decision and we now have one installer *per platform* that will install to the standard `Program files` folder, no more "x86'. 
If you update to latest, gsudo will install in the correct PF folder.
Regards!

**Reply by DervishD** [reply]
No need to be sorry, and thanks a lot for the explanation 😄 And of course, thanks a lot for `gsudo`!
